srcpkgs/pmw/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'pmw'
+pkgname=pmw
+version=5.20
+revision=1
+build_style=gnu-configure
+make_install_args="MANDIR=/usr/share/man"
```

Comment:
it doesn't seem to put any manpages to the package, is it putting it to / directly and not into $DESTDIR?

Reply:
It should be okay now